sirepo/feature_config.py

```diff
@@ -174,6 +174,22 @@ def b(msg, dev=False):
             "codes that contain proprietary information and authorization to use is granted through oauth",
         ),
         raydata=dict(
+            catalog_names=(frozenset(), set, "list of catalog names to monitor"),
+            concurrent_analyses=(
+                2,
+                int,
+                "max number of analyses that can run concurrently",
+            ),
+            db_dir=(
```

Comment:
I don't think we should hard code these paths. Use sirepo/srdb.py to get the run dir and then put them under that. They could be set pkconfig.RequiredUnlessDev which will help me remember I need to configure them in higher environments. You can also make sure they are relative paths in dev and absolute paths everywhere else.

Reply:
I'm not entirely sure I understood all this, let me know if something needs to change. e799cdc8671eaf3a2fa69713cd0682b70a4ea7bb